main/parse.c

```diff
@@ -2434,6 +2438,7 @@ extern bool parseFileWithMio (const char *const fileName, MIO *mio)
 	if (Option.printLanguage)
 	{
 		printGuessedParser (fileName, language);
+		TRACE_LEAVE();
```

Comment:
It seems that there is no associated TRACE_ENTER for this TRACE_LEAVE. It is o.k.?

What I got:

```
% ./ctags --print-language main/main.c
[>> parseFile][at 0] Parsing file main/main.c
main/main.c: C
[<< parseFileWithMio][at 0] 
[<< parseFile][at 0] 
```